app/models/immunisation_import.rb

```diff
@@ -77,7 +77,14 @@ class Row
     validates :administered, inclusion: [true, false]
     validates :delivery_method, presence: true, if: :administered
     validates :delivery_site, presence: true, if: :administered
-    validates :organisation_code, presence: true, length: { maximum: 5 }
+    validates :organisation_code,
+              presence: true,
+              length: {
+                maximum: 5
+              },
+              inclusion: {
```

Comment:
Why `inclusion` and not `comparison` with `equal_to`? Do we expect to have multiple valid ODS codes at some point?

Reply:
Ah nope, that's me not noticing `equal_to` when I read the documentation the first time!